src/app/renderer/components/common/FileTree/FileTree.tsx

```diff
@@ -36,18 +40,56 @@ export class FileTree extends React.Component<Props> {
 
     private renderFile = (file: SchemaFile): JSX.Element => {
         let className: string = '';
-        let icon: string      = Icon.FILE_O;
+        let icon: string = Icon.FILE_O;
 
+        if (file.variants && file.variants.length) {
+            icon = Icon.FILE_MULTIPLE;
+        }
+
+        file.selected = false;
```

Comment:
selected state should not be saved to the file object. It's only used here. Just create a local variable for it. like this:
```
const isSelected: boolean = !!this.props.selectedBasename && file.basename == this.props.selectedBasename;
if (isSelected) {
    className = styles.active;
    icon = Icon.FILE;
}
```

Reply:
done